waiter/src/waiter/shell_scheduler.clj

```diff
@@ -345,7 +345,7 @@
   [{:keys [:shell-scheduler/process :shell-scheduler/pid] :as instance} mem pid->memory port->reservation-atom port-grace-period-ms]
   (if (and (active? instance) (.isAlive process))
     (let [memory-allocated (* mem 1000)
```

Comment:
Should this be `1024`?

Reply:
Also, we should include the units in the names, like `-mb` and `-kb` suffixes.

Reply:
Adding the units for memory doesn't flow like adding units to time.

Good: 
 - idle-timeout-mins
 - grace-period-secs

Meh: 
 - memory-used-kib
 - memory-allocated-gib

We use mem throughout the system to mean MB.  There are no exceptions except `pid->memory`.  We could make `pid->memory` return MB but it feels weird to me.  Instead, I added a comment to the `get-pid->memory` function.